products/jbrowse-react-linear-genome-view/docs/example.md

```diff
@@ -0,0 +1,123 @@
+This is an example of an assembly and tracks configuration for a simple human
+genome example using publicly hosted data files. It includes the GRCh38
+reference sequence and a genes track.
```

Comment:
should this documentation be moved to the website?

Reply:
I'm thinking adding more documentation to the website about this will be done in #1469. This is kind of a placeholder until then.

Reply:
ok. I guess I would prefer that most documentation by default is on the website, rather than distributed throughout repo, but if needed can be here